tendrl/gluster_integration/objects/definition/gluster.yaml

```diff
@@ -60,6 +60,17 @@ namespace.gluster:
       type: Create
       uuid: b8dd2b4a-96de-4ded-b17d-374d26ec7593
       version: 1
+    SyncAggregatedClusterDetails:
+      tags:
+        - "provisioner/gluster"
+      atoms:
+        - gluster.objects.Volume.atoms.SyncRebalanceStatus
+      help: sync the rebalance status for all the volumes
```

Comment:
We would have different atoms defined for syncing utilization, connections etc later and add it here.